pkg/client/factory_test.go

```diff
@@ -204,7 +204,10 @@ func TestClientWithDatafileAccessToken(t *testing.T) {
 }
 
 func TestClientWithDefaultDecideOptions(t *testing.T) {
-	decideOptions := []decide.Options{decide.DisableDecisionEvent, decide.EnabledFlagsOnly}
+	decideOptions := decide.OptimizelyDecideOptions{
```

Comment:
I discussed with @pawels-optimizely about this change from array to struct for options. The array of options still looks good to me :) I'd like to keep this open and see how the options will be implemented in the following PRs.